src/core/ext/transport/chttp2/transport/bin_decoder.cc

```diff
@@ -147,6 +147,7 @@ bool grpc_base64_decode_partial(struct grpc_base64_decode_context* ctx) {
         case 3:
           ctx->output_cur[1] = COMPOSE_OUTPUT_BYTE_1(ctx->input_cur);
         /* fallthrough */
```

Comment:
what's the exact compiler warning you are getting here? I was under the impression that `/* fallthrough */` comment would work on most compilers. Without further evidence that this not the case we should not proceed.